tests/test_utils/test_colo_checkpoint.py

```diff
@@ -163,35 +161,37 @@ def run_checkpoint(init_spec_func, use_ddp, use_mp_reload, test_scheduler, pg):
 
     check_param_equal(model, model_reload)
 
+    if rank == 0:
+        remove('./checkpoint')
+
 
 def run_dist(rank, world_size, port, use_ddp, use_mp_reload, test_scheduler):
-    if use_ddp and world_size == 1:
-        return
-    tp_world_size = world_size // 2 if use_ddp else world_size
-    config = dict(parallel=dict(tensor=dict(mode="1d", size=tp_world_size),))
-    colossalai.launch(config=config, rank=rank, world_size=world_size, host='localhost', port=port, backend='nccl')
+    colossalai.launch(config={}, rank=rank, world_size=world_size, host='localhost', port=port, backend='nccl')
     pg = ProcessGroup(tp_degree=world_size)
-    run_checkpoint(init_1d_row_for_linear_weight_spec, use_ddp, use_mp_reload, test_scheduler=test_scheduler, pg=pg)
+    for model_name in ['bert', 'simple_net']:
+        _run_checkpoint(model_name,
+                        init_1d_row_for_linear_weight_spec,
+                        use_ddp,
+                        use_mp_reload,
+                        test_scheduler=test_scheduler,
+                        pg=pg)
 
 
 @pytest.mark.dist
 @pytest.mark.parametrize('world_size', [1, 2])
-@pytest.mark.parametrize('use_ddp', [True, False])
+@pytest.mark.parametrize('use_ddp', [False])
```

Comment:
Why not use ddp?

Reply:
ColoDDP is currently not available. @1SAA is working on DDP.